stopwatch/main.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+var start = document.querySelector('#start-btn')
+var time = document.querySelector('#timer')
+
+var sec = 0, min = 0, hour = 0
```

Comment:
Consider grouping these in an object to make them easier to pass into other functions.